forge-gui/res/cardsfolder/upcoming/vanguard_of_the_rose.txt

```diff
@@ -0,0 +1,8 @@
+Name:Vanguard of the Rose
+ManaCost:1 W
+Types:Creature Vampire Knight
+PT:3/1
+A:AB$ Pump | Cost$ Sac<1/Creature.Other;Artifact.Other/another creature or artifact> | Defined$ Self | KW$ Indestructible | SubAbility$ DBTap | SpellDescription$ CARDNAME gains indestructible until end of turn.
+SVar:DBTap:DB$ Tap | Defined$ Self | SpellDescription$ Tap it.
```

Comment:
Probably needs StackDescription$ SpellDescription
Line 5 might also need that... sometimes Pump is fine though